### ISSUE TITLE
I've found that fine-tuning doesn't work

### ISSUE THREAD

**Issue by wzl1026** [Asad-Ismail/Grounding-Dino-FineTuning#20]
Although the fine-tuning display loss is decreasing, and a lot of pth files are generated, but I found that I use different pictures, and the accuracy of the reasoning is the same, and I think the check_point.pth files obtained after training are the same, have you encountered this problem?

**Comment by HDUyiming** [comment]
Hi, I meet the same question. But the result of my test is getting worse and worse
You can take a look at my issue: https://github.com/Asad-Ismail/Grounding-Dino-FineTuning/issues/21

**Comment by wzl1026** [comment]
I think something went wrong when merging the pre-training weights and lora weights, I found that the fine-tuned detection ability is the same as the pre-trained detection ability, did you solve your problem
